<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>KD Chart 2 example: Legends</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter2" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Legends</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="legendsTV" >
          <column>
           <property name="text" >
            <string>Position</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Show Lines</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Orientation</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Alignment</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Style</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="addLegendPB" >
            <property name="text" >
             <string>A&amp;dd...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editLegendPB" >
            <property name="text" >
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeLegendPB" >
            <property name="text" >
             <string>R&amp;emove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="chartFrame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../HeaderFooter/BarChart" />
 </resources>
 <connections/>
</ui>
